### DISCUSSION TITLE
Uninstalling (maybe?) caused game breaking issues.

### DISCUSSION THREAD

**Discussion by nnelson00** [bcssov/IronyModManager#505] [Q&A]
I had installed Irony because I was running Win7 after steam support stopped (I know, I know). I could still play PDX games but I would have to bypass the launcher and therefore my only way of sorting mods. Since then I've updated to Win10 and figured the mod manager would be redundant for my purposes so I uninstalled. Since then I have been having serious problems playing any PDX games without random crashes and my PC getting bricked. I am not certain that Irony is the issue but I just want to cover all my bases.  I am also talking to PDX support to see if there is anything they can suggest but all efforts have been in vain thus far. 

How would I go about making sure all files have been properly removed?

**Comment by bcssov** [comment]
Irony is not a driver or some piece of malware to remain active after in memory once you uninstall it. Once you remove it (uninstaller or manually delete it) that is it. But if you wish to remove any files that Irony might have left (configuration files for example) you can find instructions here: https://github.com/bcssov/IronyModManager/wiki/Privacy-Policy#removing-irony
